teachertool/src/transforms/mergeEvalResult.ts

```diff
@@ -15,6 +16,10 @@ export function mergeEvalResult(criteriaInstanceId: string, outcome?: Evaluation
         newCriteriaEvalResult.result = outcome;
     }
     if (notes !== undefined) {
+        if (newCriteriaEvalResult.notes !== notes) {
```

Comment:
I'm wondering if we should do this when the outcome changes too and not notes. Not currently a scenario we have, but if we ever enabled feedback for different criteria it might become applicable.

Reply:
If we do want that, simplest approach is probably just to do this in `setEvalResult` anytime it changes, instead.